core/ui/src/main/kotlin/app/vitune/core/ui/enums/ColorPaletteName.kt

```diff
@@ -4,6 +4,5 @@ enum class ColorPaletteName(val isDynamic: Boolean) {
     Default(isDynamic = false),
     Dynamic(isDynamic = true),
     MaterialYou(isDynamic = true),
-    PureBlack(isDynamic = false),
```

Comment:
Yup, removing this could even cause a crash I guess.

Reply:
Re-added it in the fixup. Let's autosquash later